CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * [#7636](https://github.com/rubocop-hq/rubocop/issues/7636): Remove `console` from `Lint/Debugger` to prevent false positives. ([@gsamokovarov][])
 * [#7641](https://github.com/rubocop-hq/rubocop/issues/7641): **(Breaking)** Remove `Style/BracesAroundHashParameters` cop. ([@pocke][])
 * Add the method name to highlight area of `Layout/EmptyLineBetweenDefs` to help provide more context. ([@rrosenblum][])
+* [#7652](https://github.com/rubocop-hq/rubocop/pull/7652): Allow parameter name `pp` for overriding `pretty_print`. ([@masarakki][])
```

Comment:
I think it will be clearer if the cop name is specified. e.g:

```diff
-* [#7652](https://github.com/rubocop-hq/rubocop/pull/7652): Allow parameter name `pp` for overriding `pretty_print`. ([@masarakki][])
+* [#7652](https://github.com/rubocop-hq/rubocop/pull/7652): Allow `pp` to allowed names of `Naming/MethodParameterName` cop in default config. ([@masarakki][])
```